### ISSUE TITLE
Running jailbreak

### ISSUE THREAD

**Issue by mazensmz** [DarkModderVC/PS4JB#36]
Everytime new payload need to be sent, webkit hack and jailbreak need to be applied, is it required?!

**Comment by DevSoftDOTSoft** [comment]
> Everytime new payload need to be sent, webkit hack and jailbreak need to be applied, is it required?!

No. Once the Webkit Exploit is done it runs the Jailbreak auto. After that just choose your payload. Works Awesome

**Comment by mazensmz** [comment]
O understand that part, but when later decided to send new payload, I have to run the WK and JB again. Is it required?!

**Comment by DevSoftDOTSoft** [comment]
> O understand that part, but when later decided to send new payload, I have to run the WK and JB again. Is it required?!

If you leave the browser yes

**Comment by mazensmz** [comment]
So even if I make new landing page to payloaders directly, still won't work after leaving the browser?

**Comment by DevSoftDOTSoft** [comment]
> So even if I make new landing page to payloaders directly, still won't work after leaving the browser?

The WebKit is code to be able to "See" one door inside the Browser. The Kernel Exploit Will brake that door. HEN Will enter at home and unlock all the stuff. So if you leave the process will change the RAM addresses and the old "Door" will go away, and a new one will be rebuilt, so you will need to "Start Looking" again, which is Webkit...

**Comment by mazensmz** [comment]
Ha ha ha, the issue is in most the cases webkit will not find the door, and when found it, the JB will hit the wall and got crashed 🤣🤣🤣
I think  an app with ability to receive payloads will be the solution, is there anything like this?!

Thanks for the example 😊

**Comment by DevSoftDOTSoft** [comment]
> Ha ha ha, the issue is in most the cases webkit will not find the door, and when found it, the JB will hit the wall and got crashed 🤣🤣🤣
> I think an app with ability to receive payloads will be the solution, is there anything like this?!
> 
> Thanks for the example 😊

Haha No problem :)

Yha it's hard work, but that only show's how much of a masterpiece this is. The other FW's previous to 7.00 where easier to exploit.
Hum, that sound's like a netcat server... Guess there is none for 7.55 but in case you want it really bad check the FTP server and try to implement the netcat server. Guess also that it's not requested a lot, because most of user's just send all the payload's at once.
